src/main/kotlin/frc/team5499/frc2018Kotlin/path/Path.kt

```diff
@@ -0,0 +1,100 @@
+package frc.team5499.frc2018Kotlin.path
+
+import java.io.FileNotFoundException
+import java.io.FileReader
+
+import com.opencsv.CSVReader
+
+import frc.team5499.frc2018Kotlin.Constants
+import frc.team5499.frc2018Kotlin.utils.Vector2
+
+public class Path(filepath: String, backwards: Boolean = false) {
+
+    private var coordinates: MutableList<Vector2> = mutableListOf<Vector2>()
+    private var targetVelocities: MutableList<Double> = mutableListOf<Double>()
+
+    private var backwards: Boolean = false

```

Comment:
if you're defaulting `backwards` to false in the constructor, don't set it to false here. Make this line just `private var backwards: Boolean` and then set it in the `init` block

Reply:
actually, lemme double check this

Reply:
make it public as well